src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -125,12 +125,12 @@ public bool ChildContentRetrieved
         /// <summary>
         /// HTML attributes.
         /// </summary>
-        public TagHelperAttributeList HTMLAttributes { get; }
+        public TagHelperAttributeList HTMLAttributes => _htmlAttributes;
 
         /// <summary>
         /// <see cref="ITagHelper"/> bound attributes and HTML attributes.
         /// </summary>
-        public TagHelperAttributeList AllAttributes { get; }
+        public TagHelperAttributeList AllAttributes => _allAttributes;
```

Comment:
@rynowak you're misunderstanding slightly. `AllAttributes` and `HTMLAttributes` (yeah, naming issue as you mentioned in #682) remain `public` get-only properties.

@NTaylorMullen why aren't these bog-standard lazy-loaded properties? That is
``` c#
    public TagHelperAttributeList AllAttributes
    {
        get
        {
            if (_allAttributes == null)
            ...
            return _allAttributes;
        }
    }
```